### PR TITLE
Fix: Log level could not be configured from settings

### PR DESCRIPTION
**Pull request by hoh**
Solution: Add a `LOG_LEVEL` setting defaulting to "WARNING".

### CODE REVIEW

**Review by nesitor — APPROVED** [review]

**Comment by olethanh** [comment]
rebased on main

File: src/aleph/vm/orchestrator/cli.py [hunk 1]
```diff
@@ -307,6 +303,12 @@ def main():
         FAKE_DATA_PROGRAM=args.fake_data_program,
         DEBUG_ASYNCIO=args.debug_asyncio,
         FAKE_INSTANCE_BASE=args.fake_instance_base,
+        LOG_LEVEL=logging.getLevelName(args.loglevel),
+    )
+
+    logging.basicConfig(
+        level=settings.loglevel,
```
Comment (hoh) [inline src/aleph/vm/orchestrator/cli.py:310]: Typo ?
```suggestion
        level=settings.LOG_LEVEL,
```

Comment (olethanh) [reply src/aleph/vm/orchestrator/cli.py:310]: You tell me. it's your code I have just been doing rebases

**Review by olethanh — CHANGES_REQUESTED** [review]
Just tested it and it doesn't seems to work

**Comment by olethanh** [comment]
Replaced by #724 which implement the feature

**Pull request closed without merging**